src/Orleans.Core/Diagnostics/ActivityPropagationGrainCallFilter.cs

```diff
@@ -14,9 +14,17 @@ internal abstract class ActivityPropagationGrainCallFilter
         protected const string TraceStateHeaderName = "tracestate";
 
         internal const string RpcSystem = "orleans";
-        internal const string ActivitySourceName = "Microsoft.Orleans";
+        internal const string OrleansNamespacePrefix = "Orleans";
+        internal const string UserGrainActivitySourceName = "Microsoft.Orleans.User";
```

Comment:
Maybe "Application" instead of "User". It would be nice if we could do this without reflection, but I don't see an obvious mechanism for that.

Reply:
Updated